wp-multisite-waas.php

```diff
@@ -66,6 +66,25 @@
 
 require_once __DIR__ . '/inc/traits/trait-singleton.php';
 
+/**
+ * Fix for "Class 'WP_Ultimo\Database\Sites\Site_Query' not found" error
+ * Ensure all necessary database classes are loaded explicitly
+ */
+// BerlinDB core classes
+if (file_exists(__DIR__ . '/vendor/berlindb/core/src/Database/Base.php')) {
```

Comment:
Can these be move to the Sunrise class? https://github.com/superdav42/wp-multisite-waas/blob/main/inc/class-sunrise.php#L129 most of them are already here and I would expect it to fix the problem. I don't want to put things in the main plugin file unless we absolutely have to.